h/templates/layouts/base.html

```diff
@@ -5,16 +5,16 @@
       <meta charset="UTF-8" />
       <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
       <meta name="viewport" content="width=device-width,initial-scale=1" />
-      {% for attrs in meta_attrs -%}
+      {% for attrs in meta_attrs %}
         <meta {% for key, value in attrs.items() %}{{ key }}="{{ value }}" {% endfor %}/>
-      {% endfor -%}
+      {% endfor %}
+      {% for attrs in link_attrs %}
+        <link {% for key, value in attrs.items() %}{{ key }}="{{ value }}" {% endfor %}/>
+      {% endfor %}
```

Comment:
Also, not sure the links should have been moved into the `meta` block. I'd prefer to keep them with the other links.